seahorn/lib/CMakeLists.txt

```diff
@@ -24,16 +24,22 @@ target_compile_definitions(sea_proofs PUBLIC
   MAX_STRING_LEN=${MAX_STRING_LEN})
 sea_attach_bc(sea_proofs)
 
+if(SEA_ENABLE_KLEE)
+  set(SEA_SCPT ${CMAKE_SOURCE_DIR}/seahorn/scripts)
+  add_custom_target(klee_malloc ALL
+    COMMAND /bin/bash ${SEA_SCPT}/create_alloce_size_switch.sh ${SEA_LIB} ${MAX_BUFFER_SIZE})
```

Comment:
`/bin/bash` is not always available. Check what `cmake` suggests. If there is nothing generic, use `/bin/sh`

Always better to write scripts in python, we already depend on python anyways.